benchmarks/database/src/main/scala/org/renaissance/database/DbShootout.scala

```diff
@@ -14,7 +14,7 @@ import org.renaissance.License
 @Group("database")
 @Summary("Executes a shootout test using several in-memory databases.")
 @Licenses(Array(License.APACHE2))
-@SupportsJvm("18")
+@SupportsJvm("22") // Works on Java 24 in standalone mode.
```

Comment:
in standalone mode, but not in regular mode?

Reply:
Yes, in standalone mode, the `db-shootout` benchmark actually runs on Java 24, while with the harness it fails on Java 23. I suspect a classpath property issue when using a custom class loader, because Chronicle generates and compiles source code at runtime using the JDK compiler module and it fails with `ClassNotFound` exception for classes in jars that are definitely known to the custom class loader, but are not on the classpath. In standalone mode, all the jars are on the classpath, so Chronicle can find those it is looking for.

It's something to be investigated further, but I don't want it to become a blocker for the upcoming release.

Reply:
See #458